WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.m

```diff
@@ -791,15 +792,26 @@ - (void)refreshNoResultsView
         
         hideImageView = (WPDeviceIdentification.isiPhone && !isLandscape) || (WPDeviceIdentification.isiPad && isLandscape);
     }
+    NSString *image = nil;
+    NSString *subtitle = nil;
+    if (self.fetchCommentsError != nil) {
+        image = @"wp-illustration-reader-empty";
+        NSError *error = self.fetchCommentsError;
+        if (error && [error.domain isEqualToString:WordPressComRestApiErrorDomain] && error.code == WordPressComRestApiErrorCodeAuthorizationRequired) {
+            subtitle = NSLocalizedString(@"You have no access to the private blog.",
+                                          @"Error message that informs reader comments from a private blog cannot be fetched.");
+
```

Comment:
[Suggestion] I think this localized string should use the reverse dns format, right?

Reply:
I'm unsure if I can put a key, value, and description to the Objective-C code. All the `NSLocalizedString` usages I found in our app written in Objective-C look similar to this one.